### PR TITLE
Update ae.m3u

### PR DESCRIPTION
**Pull request by UltraHDR**

### CODE REVIEW

File: streams/ae.m3u [hunk 1]
```diff
@@ -149,5 +147,3 @@ https://admdn1.cdn.mangomolo.com/yastv/smil:yastv.stream.smil/playlist.m3u8
 https://weyyak-live.akamaized.net/weyyak_zee_aflam/index.m3u8
 #EXTINF:-1 tvg-id="ZeeAlwan.in",Zee Alwan (720p) [Not 24/7]
 https://weyyak-live.akamaized.net/weyyak_zee_alwan/index.m3u8
-#EXTINF:-1 tvg-id="",Takhayl (1080p)
-https://b832176c1c21d511e9ec3dc1df5d4cad.4gfbw5.channel-assembly.mediatailor.eu-west-1.amazonaws.com/v1/channel/shahid-media-prod-channel-assembly-1/index.m3u8
```
Comment (BellezaEmporium) [inline streams/ae.m3u:153]: https://shahid.mbc.net/livestream/Takhayal/livechannel-951783 but paid only

File: streams/ae.m3u [hunk 2]
```diff
@@ -63,8 +63,6 @@ https://dmisxthvll.cdn.mgmlcdn.com/dubaitvht/smil:dubaitv.stream.smil/playlist.m
 https://weyyak-live.akamaized.net/weyyak_dubai_tv/index.m3u8
 #EXTINF:-1 tvg-id="DubaiZaman.ae",Dubai Zaman (1080p) [Not 24/7]
 https://dmiffthvll.cdn.mangomolo.com/dubaizaman/smil:dubaizaman.stream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="HawasTV.ae",Hawas TV (480p)
-https://jmc-live.ercdn.net/hawastvhd/hawastvhd.m3u8
```
Comment (BellezaEmporium) [inline streams/ae.m3u:67]: works

Comment (UltraHDR) [reply streams/ae.m3u:67]: Watch it for more than a minute and it will stop. The footage hasn't updated in a long time.

**Review by freearhey — APPROVED** [review]

File: streams/ae.m3u (hunk 2 above)
Comment (BellezaEmporium) [reply streams/ae.m3u:67]: ah okay.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**